tests/unittest/_torch/auto_deploy/unit/multigpu/transformations/library/test_graph_sharding.py

```diff
@@ -28,6 +31,112 @@ def forward(self, x):
         return self.linear2(y)
 
 
+class GQABlock(nn.Module):
```

Comment:
There is a proper GQA block defined in `tests/unittest/_torch/auto_deploy/_utils_test/_model_test_utils.py`

We can just reuse it